ticketutil/bugzilla.py

```diff
@@ -149,10 +207,12 @@ def _create_ticket_request(self, params):
         """
         # Attempt to create ticket.
         try:
-            if self.token:
-                params['token'] = self.token
+            #if self.token:
+            #    params['token'] = self.token
+            params.update(self.auth.getCredentials())
```

Comment:
Made the suggested changes for _create_requests_session(), the code looks much cleaner 👍